src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -216,9 +219,25 @@ private PSVirtualNetworkGateway CreateVirtualNetworkGateway()
                 }
             }
 
-            if (this.EnableActiveActiveFeature.IsPresent && !vnetGateway.Sku.Tier.Equals(MNM.VirtualNetworkGatewaySkuTier.HighPerformance))
+            if (this.EnableActiveActiveFeature.IsPresent 
```

Comment:
These are covered by the existing tests - since this change is just adding new enums to already existing objects. There is no business logic change here.

Updated changelog md for network to reflect this.